enterprise/tomcat5/src/org/netbeans/modules/tomcat5/j2ee/JaxWsStack.java

```diff
@@ -169,21 +169,17 @@ private String[] getDetectedMetroLibs() {
 
     
     private boolean isKeystore() {
-        if (new File(catalinaHome, KEYSTORE_LOCATION).exists()) return true;
-        else return false;
+        return new File(catalinaHome, KEYSTORE_LOCATION).exists();
```

Comment:
nitpick: could be `Files.exists(Paths.get(catalinaHome, KEYSTORE_LOCATION))`

but we could do this later, going to add this pattern to the list here: https://github.com/mbien/jackpot-inspections :)

Reply:
I agree, we should do a io to nio in a new PR. Will look into that.

Reply:
this jackpot rule should cover this particular case:
```java
new java.io.File($parent, $child$).exists() :: $parent instanceof String
=> java.nio.file.Files.exists(java.nio.file.Path.of($parent, $child$))   :: sourceVersionGE(11)
=> java.nio.file.Files.exists(java.nio.file.Paths.get($parent, $child$)) :: otherwise
;;

new java.io.File($uri).exists() :: $uri instanceof java.net.URI
=> java.nio.file.Files.exists(java.nio.file.Path.of($uri))   :: sourceVersionGE(11)
=> java.nio.file.Files.exists(java.nio.file.Paths.get($uri)) :: otherwise
;;
```